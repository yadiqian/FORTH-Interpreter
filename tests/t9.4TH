what 8080 2 5 * / 700 - 100 10 1 + + 1080 10 / 900 501 + STR you CONCAT2 1010 STR 2020 STR STR CONCAT3 . CR EMIT EMIT EMIT CR .