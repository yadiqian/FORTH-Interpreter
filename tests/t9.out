20201010you1401
lol
what